src/modules/ToDos.js

```diff
@@ -3,10 +3,42 @@ import ToDo from './ToDo.js';
 export default class ToDos {
   constructor(container) {
     this.container = container;
-    this.todos = [];
+    this.todos = JSON.parse(localStorage.getItem('todos')) || [];
+    if (this.todos.length > 0) {
+      this.loadPrevious();
+    }
     this.isEditEnabled = false;
   }
 
+  loadPrevious() {
+    const instances = [];
+    const fragment = document.createDocumentFragment();
+    this.todos.forEach(({
+      description, completed, index, id,
+    }) => {
+      const instance = new ToDo(description, completed, index, id);
+      instances.push(instance);
+      const todoHtml = instance.createHtml();
+      this.addEvents(todoHtml);
+      fragment.appendChild(todoHtml);
+    });
+    this.container.appendChild(fragment);
+    this.todos = instances;
+    this.saveLocally();
+  }
+
+  saveLocally() {
+    localStorage.setItem('todos', JSON.stringify(this.todos));
+  }
+
+  removeSelected() {
+    this.todos.forEach((todo) => {
+      if (todo.completed) {
+        this.delete(this.container.querySelector(`#${todo.id}`));
+      }
+    });
+  }
+
   fixIndex() {
```

Comment:
You did a great job with the project. However, as per one of the requirements, the index of the last task in the list should be equal to the length of the array. That means the first index should start from 1 instead of 0 as in your case. ⬇️
![image](https://user-images.githubusercontent.com/33268927/213095457-00a5e311-e9d4-4ce4-b1b6-b764f49b7387.png)
![image](https://user-images.githubusercontent.com/33268927/213095609-01fa6779-54c0-49e1-b065-227a91cf2f78.png)

Kindly update your code as per the requirement.